apps-rendering/src/components/Follow/index.tsx

```diff
@@ -19,29 +19,68 @@ interface Props {
 	format: ArticleFormat;
 }
 
+const followButtonStyles: SerializedStyles = css`
+	display: flex;
+	margin: 0;
+	height: 24px;
+	width: 24px;
+	margin-right: ${space[1]}px;
+	border-radius: 100%;
+`;
+
 const styles = (format: ArticleFormat): SerializedStyles => css`
 	${textSans.small()}
-	color: ${text.follow(format)};
-	display: block;
-	padding: 0;
-	border: none;
+	color:  ${text.follow(format)};
 	background: none;
-	margin-left: 0;
+	border: none;
+	display: block;
 	margin-top: ${remSpace[1]};
-	min-height: ${remSpace[6]};
+	margin-left: 0;
 
-	svg {
-		width: ${remSpace[6]};
-		height: ${remSpace[6]};
-		fill: currentColor;
-	}
+	padding: 0;
+	text-align: left;
 
 	${darkModeCss`
-		color: ${text.followDark(format)};
+		color:  ${text.followDark(format)};
```

Comment:
I feel like the linter should have picked this up!
```suggestion
		color: ${text.followDark(format)};
```